tcks/resteasy-reactive/pom.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-tck-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-resteasy-reactive</artifactId>
+    <name>Quarkus - TCK - RestEasy Reactive</name>
+
+    <properties>
+        <exec.skip>${skipTests}</exec.skip>
+        <rrt.clone.skip>${exec.skip}</rrt.clone.skip>
+        <rrt.checkout.skip>${exec.skip}</rrt.checkout.skip>
+        <rrt.test.skip>${exec.skip}</rrt.test.skip>
+
+        <!-- careful when changing the workdir, it's referenced in update-dependencies.sh as well -->
+        <rrt.workdir>${project.build.directory}/testsuite</rrt.workdir>
+        <rrt.repo.org>quarkusio</rrt.repo.org>
+        <rrt.repo.url>https://github.com/${rrt.repo.org}/resteasy-reactive-testsuite</rrt.repo.url>
+        <!-- to avoid sudden surprises, checkout is pinned to a specific commit -->
+        <rrt.repo.ref>d7d4b999572d36a4371b75ad60427abaad2a0f9d</rrt.repo.ref>
```

Comment:
Maybe this can be in some file that can be updated automatically by some script?

Reply:
What kind of automation process do you envision? Personally, I have no problem with a script modifying a pom.xml. 😉

Reply:
Sure, that would work to. I just want to avoid looking up the latest sha and pasting it to the proper place :)

Reply:
We could also pull it up to the beginning of `properties` if that makes it easier?

Reply:
I don't think it matters - I'm going to remember one way or another :)

Reply:
I see. And I have grown a bit tired of shell scripts. 🤪
But if it makes life easier for you I can add a simple script next to the pom (and add a README of course).

Reply:
Script and readme added.